packages/p2p/src/components/payment-method-card/payment-method-card.tsx

```diff
@@ -16,34 +32,37 @@ const PaymentMethodCard = ({
     label = undefined,
     large,
     medium,
-    onClick = () => {},
-    onClickAdd = () => {},
+    onClick,
```

Comment:
can we have default values for the optional parameters rather than leaving it undefined?

Reply:
sure 👍